ee/codegen/src/__test__/nodes/generic-nodes/generic-node.test.ts

```diff
@@ -506,4 +506,40 @@ describe("GenericNode", () => {
       expect(await writer.toStringFormatted()).toMatchSnapshot();
     });
   });
+
+  describe("basic with invalid node references", () => {
+    beforeEach(async () => {
+      const nodeAttributes: NodeAttribute[] = [
+        {
+          id: "attr-1",
+          name: "invalid-node-reference",
+          value: {
+            type: "NODE_OUTPUT",
+            nodeId: "invalid-node-id",
+            nodeOutputId: "invalid-output-id",
+          },
```

Comment:
this test is testing the wrong thing - this bug has nothing to do with node output pointers.

Use the JSON found in the linear ticket as the basis for this test